src/aktualizr_lite/main.cc

```diff
@@ -11,6 +11,34 @@
 
 namespace bpo = boost::program_options;
 
+static std::shared_ptr<SotaUptaneClient> liteClient(Config &config) {
+  std::string pkey;
+  auto storage = INvStorage::newStorage(config.storage);
+  storage->importData(config.import);
+
+  EcuSerials ecu_serials;
+  if (!storage->loadEcuSerials(&ecu_serials)) {
+    // Set a "random" serial so we don't get warning messages.
+    std::string serial = config.provision.primary_ecu_serial;
+    std::string hwid = config.provision.primary_ecu_hardware_id;
+    if (hwid.empty()) {
+      hwid = Utils::getHostname();
+    }
+    if (serial.empty()) {
+      serial = Utils::getHostname();
```

Comment:
I'm not sure if I understand this logic for generating a serial. Reusing hostname seems like a problem. Serials should be unique, at least within a namespace, and hostnames are rather likely to be non-unique. Our standard technique is to use the key ID of the ECU's Uptane key.

Reply:
The problem there is that in "lite" mode there is no uptane key. I guess I could grab the UUID from the client.pem and use that since it would be unique? However, aktualizr-lite doesn't require TLS, so that might not be there either. I guess I could just generate a UUID and use that?